src/Polly.Core/Hedging/VoidHedgingHandler.cs

```diff
@@ -22,8 +22,13 @@ public sealed class VoidHedgingHandler
     /// Gets or sets the hedging action generator that creates hedged actions.
     /// </summary>
     /// <remarks>
+    /// Make sure that the action returned by the generator represents a real asynchronous work, otherwise the hedging
+    /// engine will be blocked and parallel hedged actions won't ever be executed. The generator can return a <see langword="null"/> function.
+    /// In such case the hedging is not executed for that attempt.
```

Comment:
```suggestion
    /// Make sure that the action returned by the generator represents real asynchronous work, otherwise the hedging
    /// engine will be blocked and parallel hedged actions won't ever be executed. The generator can return a <see langword="null"/> function.
    /// In such a case the hedging is not executed for that attempt.
```